modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/GetObjectRelationshipEdgeCandidatesMVCResourceCommandTest.java

```diff
@@ -258,6 +258,39 @@ private JSONArray _getObjectRelationshipEdgeCandidatesJSONArray(
 			byteArrayOutputStream.toString());
 	}
 
+	@Test
+	public void testPreventBidingWithinPublishedDefinitions() throws Exception {
+
+		ObjectDefinition objectDefinitionPublished =
```

Comment:
I believe these objectDefinitions should be renamed to:
objectDefinition1 and objectDefinition2.
Brian use to prefer using numbers to differentiate more than one instance of the same class.
I believe that he allowed the names that feliphe choose above, because in his case is too many objectDefinitions and can become confuse. But in that case, I believe is better keep the usual pattern.

It's safe certificate with @MarinhoFeliphe about this point.

Reply:
BTW, I believe the test name isn't appropriate, but I have no idea what is the most indicate. 
This point is better check with feliphe as well.